google-http-client/src/main/java/com/google/api/client/json/webtoken/JsonWebToken.java

```diff
@@ -361,6 +362,12 @@ public String toString() {
     return Objects.toStringHelper(this).add("header", header).add("payload", payload).toString();
   }
 
+  public String getRawToken() {
```

Comment:
This method needs JavaDoc. Also note that the word "raw" does not appear in the RFC. Is there a term in the RFC that does describe this?

Reply:
cloest term maybe `getEncodedToken()`